### PR TITLE
Crust grant for mobile wallet

### PR DESCRIPTION
**Pull request by duythien**
# Grant Application Checklist

- [x] The [application-template.md](https://github.com/crustio/Crust-Grants-Program/blob/main/applications/application_template.md) has been copied, renamed ( "project_name.md") and updated.
- [x] A Ethereum (USDT/CRU) address for the payment of the milestones is provided inside the application.  
- [x] The software of the project will be released under the Apache license version 2.0 as specified in the [terms and conditions](https://github.com/crustio/Crust-Grants-Program/blob/main/crust-grants-terms-and-conditions.md).
- [x] I accept the [terms and conditions](https://github.com/crustio/Crust-Grants-Program/blob/main/crust-grants-terms-and-conditions.md) and submit the PR.
- [x] The total funding amount of the project is below USDT $30k.
- [x] The initial PR contains only one commit (squash if needed before submitting your PR).
- [x] The grant will only be announced once we've successfully [delivered](https://github.com/crustio/Crust-Grant-Milestone-Delivery) the first milestone.

### CODE REVIEW

File: applications/crust-mobile-wallet.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+- Multiple: Multiple wallet in Crust
+- Easy: Receive and Send easy
+- Private & Secure: Only you can access your wallet. We don’t collect any personal data
+- Browser for DApps: Use your favourite decentralized apps & find new ones, without leaving your wallet.
+
+Below is a Video demo from "Wallet" application I have work on Hackathon gitcoin https://www.youtube.com/watch?v=VyiW7ajoRsw&ab_channel=ThienTran
+
+### Ecosystem Fit 
+As a user I want to integration to Crust network on mobile phone easy, but at the moment I have not see our system doesn't have support yet,
```
[the remark this answers is not in the document]
Comment (yyd106) [reply applications/crust-mobile-wallet.md:25]: If you are interested in doing so. You can include related works into your milestones

[the remark this answers is not in the document]
Comment (duythien) [reply applications/crust-mobile-wallet.md:25]: @yyd106 Yup. It will be implemented in Dapp

File: applications/crust-mobile-wallet.md [hunk 2]
```diff
@@ -94,16 +94,16 @@ I have development experience, including front-end development, back-end develop
...
 * **Estimated Duration:** 2 month
 * **FTE:**  1
 * **Costs:** 7,000 USD
 
 | Number | Deliverable | Specification |
 | -----: | ----------- | ------------- |
 | 1. | Staking| Adding staking app|
-| 2. | Governance| Adding governance ??|
-
+| 2. | Storage apps| integrate the storage function of Crust|
```
Comment (yyd106) [inline applications/crust-mobile-wallet.md:105]: We prefer if you have a prototype/design at first, so we can setup a meeting and talk about it

Comment (duythien) [reply applications/crust-mobile-wallet.md:105]: Give me more time try to setup design and let you know

Comment (yyd106) [reply applications/crust-mobile-wallet.md:105]: Cool, you can reach out to me through email: dean@crust.network,  if you have any questions.

**Pull request closed without merging**